packages/sdk/src/across/README.md

```diff
@@ -94,6 +94,28 @@ const { slowPct, instantPct } = await gasFeecalculator.getDepositFees(provider,
 // );
 ```
 
+#### Detailed Fees Example
+
+This uses the same underlying logic as the basic example, but returns omre information useful for display purposes.
+
+```ts
+import * as uma from "@uma/sdk"
+const { gasFeeCalculator, constants, utils } = uma.across
+
+const totalRelayed = utils.toWei(10)
+const provider = ethers.providers.getDefaultProvider()
+const umaAddress = constants.ADDRESSES.UMA
+const optionalFeeLimitPercent = 25 // this checks if fees are too high as a percentage of total amount to relay, omit to disable check
```

Comment:
Do you think we should be checking for the hard limits imposed by the contract (in addition to this)? i.e. a slow fee > 25% or fast fee > 25%?

Reply:
theres is a value `isAmountTooLow` which tells you if you are below the minimum to send. but the actual check here does the logic differently from the contract, which kind of irks me. This function adds fast+slow to be < feeLimit. Where as contract checks fast < feeLimit && slow < feeLimit. Perhaps we can re-evaluate this in a separate pr.